snuba/snuba_migrations/events_analytics_platform/0016_spans_v2.py

```diff
@@ -0,0 +1,138 @@
+from typing import List, Sequence
+
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+from snuba.migrations.operations import AddIndicesData, OperationTarget, SqlOperation
+from snuba.utils.schemas import (
+    UUID,
+    Column,
+    DateTime,
+    DateTime64,
+    Float,
+    Int,
+    Map,
+    String,
+    UInt,
+)
+
+storage_set_name = StorageSetKey.EVENTS_ANALYTICS_PLATFORM
+local_table_name = "eap_spans_2_local"
+dist_table_name = "eap_spans_2_dist"
+num_attr_buckets = 20
+
+columns: List[Column[Modifiers]] = [
+    Column("organization_id", UInt(64)),
+    Column("project_id", UInt(64)),
+    Column("service", String(Modifiers(codecs=["ZSTD(1)"]))),
+    Column("trace_id", UUID()),
+    Column("span_id", UInt(64)),
+    Column("parent_span_id", UInt(64, Modifiers(codecs=["ZSTD(1)"]))),
+    Column("segment_id", UInt(64, Modifiers(codecs=["ZSTD(1)"]))),
+    Column("segment_name", String(Modifiers(codecs=["ZSTD(1)"]))),
+    Column("is_segment", UInt(8, Modifiers(codecs=["LZ4"]))),
+    Column("_sort_timestamp", DateTime(Modifiers(codecs=["DoubleDelta", "ZSTD(1)"]))),
```

Comment:
Maybe call this `start_timestamp` and then we can have `start/end_timestamp_precise`? Unless we still think we might change what we use as the sort key?

Reply:
I don't think we'll change what we use in the sort key but maybe, instead of rounding it to the second, we'll round it to something else?

I think it's still a good idea to keep indicating it's really only meant to be used as a sort timestamp and not really treat it as the real span timestamp.